package.json

```diff
@@ -108,6 +112,7 @@
       "unicorn/filename-case": 0,
       "import/no-unassigned-import": 0,
       "import/no-extraneous-dependencies": 0,
+      "import/prefer-default-export": 0,
```

Comment:
It's actually [xo](https://github.com/sindresorhus/xo) in this case, but yes people do that.